src/nvim/ex_docmd.c

```diff
@@ -9411,3 +9411,36 @@ static void ex_terminal(exarg_T *eap)
     xfree(name);
   }
 }
+
+static void ex_plug(exarg_T *eap)
+{
+  char *path = NULL;
+  if (*eap->arg != NUL) {
+    if (os_isdir(eap->arg)) {  // full path: plug ~/.config/nvim/plugged/coolplugin
+      path = (char *)vim_strsave_fnameescape(eap->arg, FALSE);
+    } else {  // spec: plug neovim/neovimplugin
+      char_u *user_vim = vim_strsave(p_rtp);
+      strtok((char *)user_vim, ","); // split(&runtimepath, ',')[0]
+      char_u *basename = path_tail((char_u *)eap->arg); // 'neovim/neovimplugin' -> 'neovimplugin'
```

Comment:
@fmoralesc I thought that it is *user* who is supposed to call `plug` and let whatever is the current plugin/path manager do the rest, not *plugin manager* is supposed to call `plug`.

VAM will never call `plug`, especially if it is left in the current state:

1. `:plug` puts new runtimepath to the location where it is not supposed to be.
2. VAM has absolutely no reason to call `:plug` because it works fine the old way.